src/parameterizations/lateral/MOM_mixed_layer_restrat.F90

```diff
@@ -1643,9 +1683,15 @@ logical function mixedlayer_restrat_init(Time, G, GV, US, param_file, diag, CS,
              "If true, the MLE parameterization will use the mixed-layer "//&
              "depth provided by the active PBL parameterization. If false, "//&
              "MLE will estimate a MLD based on a density difference with the "//&
-             "surface using the parameter MLE_DENSITY_DIFF.", default=.false.)
-    if (.not.CS%MLE_use_PBL_MLD) call MOM_error(FATAL, "mixedlayer_restrat_init: "// &
-             "To use MLE%USE_BODNER23=True then MLE_USE_PBL_MLD must be True.")
+             "surface using the parameter MLE_DENSITY_DIFF, unless "//&
+              "BODNER_USE_MLD_003 is true.", default=.false.)
+    call get_param(param_file, mdl, "BODNER_USE_MLD_003", CS%Bodner_use_MLD_003, &
+             "If true, the Bodner parameterization will use the mixed-layer "//&
+             "depth based on MLD_003.", default=.false.)
```

Comment:
`MLE_DENSITY_DIFF` and `MLE_MLD_STRETCH` need to be defined when `BODNER_USE_MLD_003=True`.

```
if (CS%Bodner_use_MLD_003) then
      call get_param(param_file, mdl, "MLE_DENSITY_DIFF", CS%MLE_density_diff, &
           "Density difference used to detect the mixed-layer "//&
           "depth used for the mixed-layer eddy parameterization "//&
           "by Fox-Kemper et al. (2010)", units="kg/m3", default=0.03, scale=US%kg_m3_to_R)
      call get_param(param_file, mdl, "MLE_MLD_STRETCH", CS%MLE_MLD_stretch, &
             "A scaling coefficient for stretching/shrinking the MLD "//&
             "used in the MLE scheme. This simply multiplies MLD wherever used.",&
             units="nondim", default=1.0)
    endif
```